Framework/CurveFitting/src/FuncMinimizers/FABADAMinimizer.cpp

```diff
@@ -467,6 +467,10 @@ void FABADAMinimizer::tieApplication(const size_t &parameterIndex,
     m_fitFunction->setParameter(parameterIndex, newValue);
   }
 
+  // Apply fix to the parameter if it is fixed
+  if (m_fitFunction->isFixed(parameterIndex))
+    applyFix(parameterIndex, newParameters);
```

Comment:
Why cannot a fixed parameter be skipped?

Reply:
I think I agree, skipping fixed parameters may be a better way of doing this! 👍